### PR TITLE
Extending type coverage for vector in vector_api tests

### PR DESCRIPTION
**Pull request by kopylovanat**
Depends on PR https://github.com/KhronosGroup/SYCL-CTS/pull/79 (1 commit)
Only commits start with https://github.com/KhronosGroup/SYCL-CTS/commit/95e78c90740197bb3e51e6b258a91eacc9e2168c for this PR

### CODE REVIEW

**Review by ProGTX — APPROVED** [review]

File: tests/common/common_python_vec.py [hunk 1]
```diff
@@ -333,12 +377,334 @@ def write_source_file(test_str, func_calls, test_name, input_file, output_file,
...
 
+    source = get_ifdef_string(source, type_str)
+
     with open(output_file, 'w+') as output:
         output.write(source)
+
+def get_standard_and_fixed_width_types():
+    types = list()
+    types.append('char')
+    types.append('cl::sycl::byte')
```
Comment (keryell) [inline tests/common/common_python_vec.py:393]: Curious. Why not just creating the list with these 2 values?

File: tests/common/vector.template [hunk 2]
```diff
@@ -38,3 +39,4 @@ class TEST_NAME : public util::test_base {
 util::test_proxy<TEST_NAME> proxy;
 
 } /* namespace vector_$CATEGORY_$TYPE_NAME__ */
+$ENDIF
```
Comment (keryell) [inline tests/common/vector.template:42]: Missing end-of-line.

File: tests/common/vector_swizzles.template [hunk 3]
```diff
@@ -69,3 +69,4 @@ class TEST_NAME : public util::test_base {
 util::test_proxy<TEST_NAME> proxy;
 
 } /* namespace vector_swizzles_$TYPE_NAME__ */
+$ENDIF
```
Comment (keryell) [inline tests/common/vector_swizzles.template:72]: Missing end-of-line.

File: tests/common/vector.template (hunk 2 above)
Comment (kopylovanat) [reply tests/common/vector.template]: Fixed in PR #79

File: tests/common/vector_swizzles.template (hunk 3 above)
Comment (kopylovanat) [reply tests/common/vector_swizzles.template]: Fixed in PR #79

File: tests/common/common_python_vec.py (hunk 1 above)
Comment (kopylovanat) [reply tests/common/common_python_vec.py]: Yes, this is better, thank you. Changed to creating a list with them.

**Review by keryell — APPROVED** [review]
That looks good

**Pull request merged**